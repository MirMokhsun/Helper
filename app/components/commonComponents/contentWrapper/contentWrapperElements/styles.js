import { StyleSheet } from 'react-native';

export const styles = StyleSheet.create({
    container: {
        marginVertical: 20,
        justifyContent: "center",
        alignItems: 'center',
    },
    text: {
        marginVertical: 10,
        lineHeight: 30,
        fontWeight: 'bold',
        fontFamily: "Roboto-Light",
        fontSize: 22,
        textAlign: 'center',
        color: '#000756',
    }
});