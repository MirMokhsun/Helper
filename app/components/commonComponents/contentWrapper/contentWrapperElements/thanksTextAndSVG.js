import React from 'react';
import { View, Text } from 'react-native';
import Thanks from '../../svg/Thanks';
import { styles } from './styles';

export class ThanksTextAndSVG extends React.PureComponent {
    constructor(props) {
        super(props)
    }

    render() {
        const { text } = this.props;
        return (
            <View style={styles.container}>
                <Thanks />
                {text ? <Text style={styles.text}>{text}</Text> : null}
            </View>
        );
    }
}