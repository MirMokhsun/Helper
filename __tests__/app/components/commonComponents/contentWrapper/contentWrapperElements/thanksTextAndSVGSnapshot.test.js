import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { ThanksTextAndSVG } from '../../../../../../app/components/commonComponents/contentWrapper/contentWrapperElements/thanksTextAndSVG';

describe('ThanksTextAndSVG', () => {
    it('create snapshot ThanksTextAndSVG with text', () => {
        const renderer = new ShallowRenderer()
        let toast = renderer.render(<ThanksTextAndSVG text={'text'} />);
        expect(toast).toMatchSnapshot();
    });

    it('create snapshot ThanksTextAndSVG', () => {
        const renderer = new ShallowRenderer()
        let toast = renderer.render(<ThanksTextAndSVG />);
        expect(toast).toMatchSnapshot();
    });
});