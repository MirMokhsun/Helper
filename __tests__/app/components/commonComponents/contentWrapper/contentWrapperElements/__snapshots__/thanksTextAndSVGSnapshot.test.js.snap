// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThanksTextAndSVG create snapshot ThanksTextAndSVG 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "justifyContent": "center",
      "marginVertical": 20,
    }
  }
>
  <Thanks />
</View>
`;

exports[`ThanksTextAndSVG create snapshot ThanksTextAndSVG with text 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "justifyContent": "center",
      "marginVertical": 20,
    }
  }
>
  <Thanks />
  <Text
    style={
      Object {
        "color": "#000756",
        "fontFamily": "Roboto-Light",
        "fontSize": 22,
        "fontWeight": "bold",
        "lineHeight": 30,
        "marginVertical": 10,
        "textAlign": "center",
      }
    }
  >
    text
  </Text>
</View>
`;
